src/NodaTime/Text/InstantPattern.cs

```diff
@@ -189,6 +196,14 @@ private static InstantPattern Create(string patternText, NodaFormatInfo formatIn
         /// <param name="newTemplateValue">The template value for the new pattern, used to fill in unspecified fields.</param>
         /// <returns>A new pattern with the given template value.</returns>
         public InstantPattern WithTemplateValue(Instant newTemplateValue) =>
-            Create(PatternText, FormatInfo, newTemplateValue);
+            Create(PatternText, FormatInfo, newTemplateValue, TwoDigitYearMax);
+
+        /// <summary>
+        /// Creates a pattern like this one, but with a different <see cref="TwoDigitYearMax"/> value.
+        /// </summary>
+        /// <param name="twoDigitYearMax">The value to use for <see cref="TwoDigitYearMax"/> in the new pattern.</param>
```

Comment:
worth adding something like "The value (0-99) to use..." to make the range clear?

Separately, where do we document the default values for this type? Probably we should document the default value for this new property?

Reply:
Done. Will add the default into the description of the property.